package.json

```diff
@@ -35,6 +35,7 @@
 		"jsonschema": "^1.4.0",
 		"minimatch": "^3.1.2",
 		"mkdirp": "^1.0.4",
+		"repo-report": "github:KhatiaIvanova/repo-report",
```

Comment:
maybe you ran `npm install .`?
```suggestion
```

Reply:
@ljharb  
I will update those lines, but I Ihave a question. What we did for issue 80 is Console.log while loading repos from Github, after await will be done Console.log loaded. 

I don't think that Console. clear after await will be done is a good solution because in this case, it will clear the whole console, which is bad. and removing only the last log is not possible, due to limitations.

Also, it would be better to change the name of this pull request for now, after we are not working on a code of conduct for now. can I change it by myself?